client/src/modules/login/login.html

```diff
@@ -40,5 +40,7 @@
     <div class="login-other-container-item flex flex-center">
       <a ui-sref="signup">Don't have an account? Sign up today</a>
     </div>
+    <br/>
```

Comment:
don't use br's, use margin, more easily adjustable - also I think just a little bit more spacing than is there right now